src/components/Modals/Onboarding/Onboarding.vue

```diff
@@ -155,7 +152,29 @@ export default defineComponent({
       this.currentPage = page;
     },
     goNext() {
-      this.currentPage = this.currentPage === FINAL_PAGE ? FINAL_PAGE : this.currentPage + 1;
+      // Onboarding Basic
+      if (this.currentPage === 1) {
+        // Display error if a required field is empty
+        // Display error if onboarding data includes a major or minor that doesn't exist in requirementsJSON
+        this.isError =
+          this.name.firstName === '' ||
+          this.name.lastName === '' ||
+          this.onboarding.college === '' ||
+          this.onboarding.gradYear === '' ||
+          this.onboarding.entranceYear === '';
+
+        this.isInvalidMajorOrMinorError =
+          this.onboarding.major
+            .map(getMajorFullName)
+            .some((majorFullName: string) => majorFullName === '') ||
+          this.onboarding.minor
+            .map(getMinorFullName)
+            .some((minorFullName: string) => minorFullName === '');
```

Comment:
Why don't we put this as computed property, so that the warning can auto-showup in button before clicking next

Reply:
Yes good call! I will push those changes